index.html

```diff
@@ -18,12 +18,23 @@
       href="https://fonts.googleapis.com/css2?family=Playfair+Display&display=swap"
       rel="stylesheet"
     />
-    <title>Portifolio</title>
+    <!-- Global site tag (gtag.js) - Google Analytics -->
```

Comment:
- [ ] Please make sure to follow all of the requirements asked by the last reviewer. These include updating the description of this pull request to include a report on every one of the accessibility checks you've done and links to the lines where you made changes to improve it.

If you unsure what this means, make sure to read carefully [the requirements of this milestone here.](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/6_accessability.md)

![image](https://user-images.githubusercontent.com/79114233/139929222-5e305d6a-f1b0-4393-b61e-37dfa00f6a51.png)

Reply:
I change it this is the index.html file for deploy not the original code my index.html file have the title Meqdam's portfolio